src/modules/ens/ens.service.ts

```diff
@@ -21,8 +21,12 @@ import {
   DomainHierarchy,
   ResolverContractType,
   DomainNotifier__factory,
+  StakingPoolFactory__factory,
+  VOLTA_STAKING_POOL_FACTORY_ADDRESS
 } from '@energyweb/iam-contracts';
 import type { DomainNotifier } from "@energyweb/iam-contracts/dist/ethers-v4/DomainNotifier";
+import { StakingService } from '../staking/staking.service';
+import { StakingPoolFactory } from '@energyweb/iam-contracts/dist/ethers-v4/StakingPoolFactory';
```

Comment:
Why is Staking being mixed in with the ENS code? In my opinion, it is important to try to keep this functionality separate. Ideally, it would be possible to split out a "staking" cache service from the "ens" cache service, IMO.